parser-typechecker/src/Unison/Codebase/FileCodebase.hs

```diff
@@ -157,18 +158,12 @@ initCodebase dir = do
 -- get the codebase in dir, or in the home directory if not provided.
 getCodebaseOrExit :: Maybe FilePath -> IO (Codebase IO Symbol Ann)
 getCodebaseOrExit mdir = do
-  (dir, errMsg) <- case mdir of
-    Just dir -> do
-      dir' <- P.string <$> canonicalizePath dir
-      let errMsg = P.lines ["No codebase exists in " <> dir'
-                           , "Run `ucm -codebase " <> P.string dir <> " init` to create one, then try again!"]
-      pure ( dir, errMsg)
-    Nothing -> do
-      dir <- getHomeDirectory
-      let errMsg = P.lines [ "No codebase exists in " <> P.string dir
-                           , "Run `ucm init` to create one, then try again!" ]
-      pure (dir, errMsg)
-
+  dir <- getCodebaseDir mdir
+  prettyDir <- P.string <$> canonicalizePath dir
+  let errMsg = P.lines
+        [ "No codebase exists in " <> prettyDir
+        , "Run `ucm -codebase " <> prettyDir
+          <> " init` to create one, then try again!"]
```

Comment:
Refactor to make use of the existing `getCodebaseDir` which already looks up the user's home directory.